static/docs/get-started/add-files.md

```diff
@@ -2,24 +2,21 @@
 
 DVC allows storing and versioning data files, ML models, directories,
 intermediate results with Git, without checking the file contents into Git.
-Let's get a sample dataset to play with:
+Let's get a dataset example to play with:
 
 ```dvc
 $ mkdir data
-$ wget https://data.dvc.org/get-started/data.xml -O data/data.xml
+$ cd data
+dvc import https://github.com/iterative/dataset-registry \
```

Comment:
the problem with this is that we don't need `dvc add` after that, here we can use `dvc get` and clarify that it's similar to `wget` but allows to use Git repo as a source ... for import we need to come with a separate section or use it in some other place ...